Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -722,6 +776,9 @@ def tc_get_indicator_command():
         if include_observations:
             if 'IndicatorsObservations' in indicators[0]:
                 del indicators[0]['IndicatorsObservations']
+        if include_attributes:
+            if 'IndicatorAttributes' in indicators[0]:
+                del indicators[0]['IndicatorAttributes']
```

Comment:
Remove IndicatorAttributes from the Markdown to coincide with the styling of the rest of the function